### ISSUE TITLE
Write tests for all the apps 

### ISSUE THREAD

**Issue by DESHRAJ** [ncs-jss/HTTP_200#54]

**Comment by insomniac12** [comment]
@DESHRAJ can I work on this ?

**Comment by shriyanka** [comment]
@insomniac12  Sure you can always pick the task.  :) 
@DESHRAJ  assigning it to her if its okay :)

**Comment by shriyanka** [comment]
@insomniac12  sure you can always pick something :)
@DESHRAJ  assigning it to @insomniac12 if that's okay :)

**Comment by DESHRAJ** [comment]
Yeah sure. Go ahead. Let me know if you need sample tests from my side so that can help you.

**Comment by insomniac12** [comment]
@DESHRAJ yes ,I need the sample tests !!

**Comment by shriyanka** [comment]
@insomniac12 
For time being you can refer this link for basic understanding of writing Tests in django:
https://docs.djangoproject.com/en/1.9/topics/testing/overview/
http://toastdriven.com/blog/2011/apr/10/guide-to-testing-in-django/
https://realpython.com/blog/python/testing-in-django-part-1-best-practices-and-examples/
rest @DESHRAJ  will provide you with samples.

**Comment by insomniac12** [comment]
@shriyanka 
Thanks.I have read these all . I want to know if there is anything specific or different  to be done for the extended user model ??

**Comment by DESHRAJ** [comment]
@insomniac12 see this http://stackoverflow.com/questions/15073227/django-unit-test-simple-example 

You can get started with this for now. I will write some tests as I get time. You should start writing and create a PR so that we can see and make changes as required.
